contract/src/jar/model/common.rs

```diff
@@ -210,209 +41,32 @@ pub struct JarCache {
 }
 
 impl Contract {
-    pub(crate) fn create_jar(
+    pub(crate) fn deposit(
         &mut self,
         account_id: AccountId,
         ticket: JarTicket,
         amount: U128,
```

Comment:
[nit]
Let's change to `u128` here. It is not an interface method and will be easier to use it in test.
```suggestion
        amount: u128
```

Reply:
Done